cli/lsp/code_lens.rs

```diff
@@ -220,9 +220,10 @@ async fn resolve_implementation_code_lens(
   data: CodeLensData,
   language_server: &mut language_server::Inner,
 ) -> Result<lsp::CodeLens, AnyError> {
-  let line_index = language_server
-    .get_line_index_sync(&data.specifier)
+  let asset_or_doc = language_server
+    .get_cached_asset_or_document(&data.specifier)
     .unwrap();
+  let line_index = asset_or_doc.line_index();
```

Comment:
Although this `.unwrap()` was here before, it feels like we should not risk a panic, and instead return a err if for whatever reason the document is missing.